xmtp_mls/src/groups/mod.rs

```diff
@@ -446,73 +453,85 @@ impl MlsGroup {
         Ok(messages)
     }
 
+    /**
+     * Add members to the group by account address
+     *
+     * If any existing members have new installations that have not been added, the missing installations
+     * will be added as part of this process as well.
+     */
     pub async fn add_members<ApiClient>(
         &self,
-        account_addresses_to_add: Vec<String>,
         client: &Client<ApiClient>,
+        account_addresses_to_add: Vec<String>,
     ) -> Result<(), GroupError>
     where
         ApiClient: XmtpApi,
     {
         let account_addresses = sanitize_evm_addresses(account_addresses_to_add)?;
+        let inbox_id_map = client.api_client.get_inbox_ids(account_addresses).await?;
         // get current number of users in group
         let member_count = self.members()?.len();
-        if member_count + account_addresses.len() > MAX_GROUP_SIZE as usize {
+        if member_count + inbox_id_map.len() > MAX_GROUP_SIZE as usize {
```

Comment:
Could multiple addresses in `account_addresses` link to the same inbox id so that we over-calculate the group size?

Reply:
Only until we update `group.members()` to be based on inbox ID. Coming very soon